taichi/backends/cpu/codegen_cpu.cpp

```diff
@@ -29,32 +29,88 @@ class CodeGenLLVMCPU : public CodeGenLLVM {
       step = -1;
     }
 
-    auto *tls_prologue = create_xlogue(stmt->tls_prologue);
-
-    // The loop body
-    llvm::Function *body;
-    {
-      auto guard = get_function_creation_guard(
-          {llvm::PointerType::get(get_runtime_type("Context"), 0),
-           llvm::Type::getInt8PtrTy(*llvm_context),
-           tlctx->get_data_type<int>()});
+    if (prog->config.arch == Arch::wasm) {
+      using namespace llvm;
```

Comment:
Remove `using namespace llvm`. Let's use `auto*` if it's redundant to type `llvm::BasicBlock` twice.

Reply:
Done